### PR TITLE
Jakarta Standard Tag Library 1.2 Spec

### PR DESCRIPTION
**Pull request by ren-zhijun-oracle**
Signed-off-by: Zhijun Ren <ren.zhijun@oracle.com>

## Specification PR template
When creating a specification project release review, create two PRs with the content divided as follows.

Include the following in PR#1:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat_x.y.pdf
- [x] Specification HTML in the form of wombat_x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Release Review, a summary that a Compatible Implementation
      is complete, passes the TCK, and that the TCK includes sufficient
      coverage of the specification. The TCK users guide MUST include
      the instructions to run the compatible implementations used to
      validate the release.  Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://oss.sonatype.org/content/repositories/staging/jakarta/servlet/jsp/jstl/jakarta.servlet.jsp.jstl-api/1.2.7/  
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
     http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-tags-tck-1.2.0.zip
- [x] The URL of the compatibility certification request issue:
     https://github.com/eclipse-ee4j/jstl-api/issues/35

Include the following in PR#2:
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory.

### CODE REVIEW

**Comment by dblevins** [comment]
This specification PDF does not look right.  There's no cover page, logo, specification name or version and the footer says the status is "draft"

**Comment by dblevins** [comment]
The new PDF looks beautiful.

Gentle note that your checkbox for OSSRH staging repository URL should not be for Maven Central, but a staged an closed, but unreleased, location like this:

 - https://oss.sonatype.org/content/repositories/jakartaejb-1013/jakarta/ejb/jakarta.ejb-api/3.2.6/

The 'release' button should only be clicked once the Specification Committee has voted, which hasn't happened yet.

Simple mistake and entirely forgivable.  Just FYI for next time.

**Comment by bshannon** [comment]
> Gentle note that your checkbox for OSSRH staging repository URL should not be for Maven Central, but a staged an closed, but unreleased, location like this:
> 
>     * https://oss.sonatype.org/content/repositories/jakartaejb-1013/jakarta/ejb/jakarta.ejb-api/3.2.6/

No, it's not supposed to be in a "local" staging repository, it's supposed to be in the "global" staging repository, e.g., https://oss.sonatype.org/content/repositories/staging/jakarta/ejb/jakarta.ejb-api/3.2.6/, which it is.

**Comment by dblevins** [comment]
This spec has been scheduled for review and promotion to vote August 21st.  Current checklist below for convenience.
https://github.com/jakartaee/specification-committee/blob/master/spec_review_checklist.md

**Comment by kwsutter** [comment]
# Spec Review Checklist

1. Spec PR
  - [ ] PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
  - [ ] Directory of form {spec}/x.y
  - [ ] PDF of form {spec}-spec-x.y.pdf ("-spec" preferred but not required)
  - [ ] HTML of form {spec}-spec-x.y.html ("-spec" preferred but not required)
  - [ ] Index page {spec}/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
  - [ ] Index page {spec}/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_index_template.md)
  - [ ] No other files (e.g., no jakarta_ee_logo_schooner_color_stacked_default.png)
  - [ ] Staging repository link of the form https://oss.sonatype.org/conte/repositories/staging/jakarta/{spec}/jakarta-{spec}-api/x.y.z/
  - [ ] EFTL TCK link of the form http://download.eclipse.org/.../+.zip
  - [ ] Compatibility certification link of the form https://github.com/eclipse-ee4j/{project}/#{issue}
  - [ ] Second PR for just apidocs

2. _index.md
  - [ ] Link to spec pdf
  - [ ] Link to spec html
  - [ ] Link to apidocs
  - [ ] Link to final TCK download zip file of the form https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip
  - [ ] Link to API jar file of the form https://search.maven.org/artifact/jakarta.{spec}/jakarta.{spec}-api/x.y.z/jar
  - [ ] Name of and link to at least one Compatible Implementation

3. javadocs
  - [ ] Footer contains Eclipse copyright and link to license
  - [ ] ESFL license is included, usually as doc-files/speclicense.html
  - [ ] no META-INF directory in PR
  - [ ] javadocs-jar artifact matches apidocs (optional for this release)

4. Spec PDF
  - [ ] Correct spec title
  - [ ] Version number of the form x.y, not x.y.z
  - [ ] Correct Eclipse copyright line
  - [ ] No DRAFT or SNAPSHOT
  - [ ] Correct Logo

5. Spec HTML
  - [ ] Same as PDF

6. TCK zip file
  - [ ] README file (optional for this release)
  - [ ] EFTL license file, preferably named LICENSE.md
  - [ ] User's Guide (or equivalent documentation)
  - [ ] How to test the Compatible Implementation(s) listed in _index.md above with the TCK (may be in UG)

7. TCK User's Guide (or equivalent documentation)
  - [ ] Software requirements listed
  - [ ] Installation and configuration described
  - [ ] How to run tests
  - [ ] Where to file challenges

8. Compatibility certification request
  - [ ] Request follows [template](https://github.com/jakartaee/specification-committee/blob/master/compatibility-certification-request.md)
  - [ ] SHA-256 fingerprint matches staged TCK zip file
  - [ ] Request issue has `certification` label.

9. TCK results summary
  - [ ] Page is hosted by Compatible Implementation project
  - [ ] Includes all information from certification request
  - [ ] Summary includes number of tests passed, failed, errors
  - [ ] SHA-256 fingerprint matches staged TCK zip file on cert request

**Comment by NottyCode** [comment]
# Spec Review Checklist

1. Spec PR
  - [x] PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
  - [x] Directory of form {spec}/x.y
  - [ ] PDF of form {spec}-spec-x.y.pdf ("-spec" preferred but not required)
  - [ ] HTML of form {spec}-spec-x.y.html ("-spec" preferred but not required)
  - [x] Index page {spec}/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
  - [x] Index page {spec}/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_index_template.md)
  - [x] No other files (e.g., no jakarta_ee_logo_schooner_color_stacked_default.png)
  - [ ] Staging repository link of the form https://oss.sonatype.org/conte/repositories/staging/jakarta/{spec}/jakarta-{spec}-api/x.y.z/
  - [x] EFTL TCK link of the form http://download.eclipse.org/.../+.zip
  - [x] Compatibility certification link of the form https://github.com/eclipse-ee4j/{project}/#{issue}
  - [x] Second PR for just apidocs

2. _index.md
  - [x] Link to spec pdf
  - [x] Link to spec html
  - [x] Link to apidocs
  - [ ] Link to final TCK download zip file of the form https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip
  - [x] Link to API jar file of the form https://search.maven.org/artifact/jakarta.{spec}/jakarta.{spec}-api/x.y.z/jar
  - [ ] Name of and link to at least one Compatible Implementation

3. javadocs
  - [ ] Footer contains Eclipse copyright and link to license
  - [x] ESFL license is included, usually as doc-files/speclicense.html
  - [x] no META-INF directory in PR
  - [x] javadocs-jar artifact matches apidocs (optional for this release)

4. Spec PDF
  - [x] Correct spec title
  - [x] Version number of the form x.y, not x.y.z
  - [x] Correct Eclipse copyright line
  - [x] No DRAFT or SNAPSHOT
  - [x] Correct Logo

5. Spec HTML
  - [x] Same as PDF

6. TCK zip file
  - [ ] README file (optional for this release)
  - [x] EFTL license file, preferably named LICENSE.md
  - [x] User's Guide (or equivalent documentation)
  - [ ] How to test the Compatible Implementation(s) listed in _index.md above with the TCK (may be in UG)

7. TCK User's Guide (or equivalent documentation)
  - [x] Software requirements listed
  - [x] Installation and configuration described
  - [x] How to run tests
  - [x] Where to file challenges

8. Compatibility certification request
  - [x] Request follows [template](https://github.com/jakartaee/specification-committee/blob/master/compatibility-certification-request.md)
  - [x] SHA-256 fingerprint matches staged TCK zip file
  - [x] Request issue has `certification` label.

9. TCK results summary
  - [x] Page is hosted by Compatible Implementation project
  - [x] Includes all information from certification request
  - [x] Summary includes number of tests passed, failed, errors
  - [x] SHA-256 fingerprint matches staged TCK zip file on cert request

**Comment by NottyCode** [comment]
The following issues were hit running the checklist:

1. The pdf and html use _ rather than - between the spec and version. It is preferred to have -spec.
1. Links to maven central rather than a staging site (as reported earlier)
1. The TCK link in the PR is to a file `eclipse-tags-tck-1.2.0.zip` and I'd expect `eclipse-jstl-tck-1.2.0.zip` Also when you extract the zip the top directory is `tags-tck` rather than `jstl-tck`
1. The link in _index.md to the TCK uses stl rather than jstl in the directory for the spec: http://downloads.eclipse.org/jakarta/stl/1.2/jstl_tck_1.2.zip
1. There is no link in _index.md to the compatible implementation, just the name.
1. All the other javdoc PR's that I recall looking at had a copyright year of 2019, and this has 2017-2019. I want to check with the spec committee if this is ok. I'm used to 2017, 2019 for multiple years rather than -. There is also no link to the EFTL in the footer as there is for other Javadoc.

**Comment by NottyCode** [comment]
@ren-zhijun-oracle as I understand the requirements the link in _index.md should be to the final location the TCK will reside once this is approved. The update moved it to the staging location. I believe the url would be:

    https://download.eclipse.org/jakartaee/jstl/1.2/jstl-tck-1.2.0.zip

I've updated the checklist and my earlier comment to indicate what has been addressed.

File: jstl/1.2/_index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Jakarta Standard Tag Library Spec 1.2"
+date: 2019-07-18T16:10:38-04:00
```
Comment (bshannon) [inline jstl/1.2/_index.md]: The time component can be removed.

File: jstl/1.2/_index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+---
+Jakarta Standard Tag Library encapsulates as simple tags the core functionality common to many Web applications.
+Jakarta Standard Tag Library has support for common, structural tasks such as iteration and conditionals, tags
+for manipulating XML documents, internationalization tags, and SQL tags. It also provides a framework for integrating
+existing custom tags with Jakarta Standard Tag Library tags.
+
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.pdf) (PDF)
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.html) (HTML)
+* [Jakarta Standard Tag Library 1.2 Javadoc](./apidocs)
+* [Jakarta Standard Tag TCK](http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8/promoted/tags-tck-1.2.0.zip)
```
Comment (bshannon) [inline jstl/1.2/_index.md]: This should be a link to the final download location:
https://download.eclipse.org/jakartaee/tags/1.2/eclipse-tags-tck-1.2.0.zip

File: jstl/1.2/_index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+summary: ""
+---
+Jakarta Standard Tag Library encapsulates as simple tags the core functionality common to many Web applications.
+Jakarta Standard Tag Library has support for common, structural tasks such as iteration and conditionals, tags
+for manipulating XML documents, internationalization tags, and SQL tags. It also provides a framework for integrating
+existing custom tags with Jakarta Standard Tag Library tags.
+
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.pdf) (PDF)
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.html) (HTML)
+* [Jakarta Standard Tag Library 1.2 Javadoc](./apidocs)
```
Comment (bshannon) [inline jstl/1.2/_index.md]: An entry needs to be added after this one for the tag documentation, which should be included
in the javadocs PR, perhaps as a "tagdocs" directory or something similar.

File: jstl/1.2/_index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+|IBM                    |         |         |          |
+|Oracle                 |         |         |          |
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
+## Plan Review
```
Comment (bshannon) [inline jstl/1.2/_index.md]: The Plan Review section should be removed.

File: jstl/1.2/_index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Jakarta Standard Tag Library Spec 1.2"
+date: 2019-07-18T16:10:38-04:00 
+summary: ""
```
Comment (bshannon) [inline jstl/1.2/_index.md]: The summary should be "First release for Jakarta EE 8".

**Comment by NottyCode** [comment]
In my prior comment I said the TCK has the wrong name having tags rather than jstl. On the spec committee mail list @bshannon pointed out that tags was the eclipse agreed code for the spec. 

There is a email thread about tags vs jstl which @dblevins and @bshannon are coming down on the side of tags, this would mean the TCK is right and everything else is wrong. I’ll leave it until after the weekend for others to give their view before updating the checklist based on this, but I’d suggest holding off on any TCK changes with the expectation that tags is right and jstl is wrong.

File: jstl/1.2/_index.md (hunk 3 above)
Comment (ren-zhijun-oracle) [reply jstl/1.2/_index.md]: Hi Bill, can you explain more clear about this comment?

Comment (ren-zhijun-oracle) [reply jstl/1.2/_index.md]: Hi Bill, can you make it more clear to me? where can get or generate the tag docs under "tagdocs" directory?

**Comment by keilw** [comment]
This seems like another source of confusion and acronym forking. If "tags" was the desired short form, then why wasn't the name properly adjusted to something like "Jakarta Standard Tags"? Similar to what was done e.g. with "Jakarta Messaging"

File: jstl/1.2/_index.md (hunk 1 above)
Comment (bshannon) [reply jstl/1.2/_index.md]: You removed both the date and the time.  We still need the date: header.
date: 2019-07-18

File: jstl/1.2/_index.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+---
+Jakarta Standard Tag Library encapsulates as simple tags the core functionality common to many Web applications.
+Jakarta Standard Tag Library has support for common, structural tasks such as iteration and conditionals, tags
+for manipulating XML documents, internationalization tags, and SQL tags. It also provides a framework for integrating
+existing custom tags with Jakarta Standard Tag Library tags.
+
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.pdf) (PDF)
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.html) (HTML)
+* [Jakarta Standard Tag Library 1.2 Javadoc](./apidocs)
+* [Jakarta Standard Tag TCK](https://download.eclipse.org/jakartaee/jstl/1.2/jstl-tck-1.2.0.zip)
```
Comment (bshannon) [inline jstl/1.2/_index.md]: "Tag TCK" -> "Tag Library TCK"

The URL should be:
https://download.eclipse.org/jakartaee/tags/1.2/eclipse-tags-tck-1.2.0.zip

File: jstl/1.2/_index.md (hunk 3 above)
Comment (ren-zhijun-oracle) [reply jstl/1.2/_index.md]: Hi Bill, fixed, please help to check.

File: tags/1.2/_index.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+for manipulating XML documents, internationalization tags, and SQL tags. It also provides a framework for integrating
+existing custom tags with Jakarta Standard Tag Library tags.
+
+* [Jakarta Standard Tag Library 1.2 Specification Document](./tags-1.2-spec.pdf) (PDF)
+* [Jakarta Standard Tag Library 1.2 Specification Document](./tags-1.2-spec.html) (HTML)
+* [Jakarta Standard Tag Library 1.2 Javadoc](./apidocs)
+* [Jakarta Standard Tag Library 1.2 Tagdoc](./tagdocs)
+* [Jakarta Standard Tag Library TCK](https://download.eclipse.org/jakartaee/tags/1.2/eclipse-tags-tck-1.2.0.zip)
+* Maven coordinates
+  * [jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api:jar:1.2](https://search.maven.org/artifact/jakarta.servlet.jsp.jstl/jakarta.servlet.jsp.jstl-api/1.2/jar)
```
Comment (kwsutter) [inline tags/1.2/_index.md]: Shouldn't this be version 1.2.5 or 1.2.7 (not sure what happened to 1.2.6):  https://oss.sonatype.org/content/repositories/staging/jakarta/servlet/jsp/jstl/jakarta.servlet.jsp.jstl-api/

Comment (starksm64) [reply tags/1.2/_index.md]: It should be 1.2.5 as the Jakarta EE 8 release with jakarta coords was the 1.2.4 release.

**Comment by kwsutter** [comment]
@ren-zhijun-oracle Could you also file a PR for updating jakartaee-api pom.xml with the proper Jakarta JSTL version (1.2.5 or 1.2.7, per the earlier comment)?

File: tags/1.2/_index.md (hunk 7 above)
Comment (kwsutter) [reply tags/1.2/_index.md]: We decided on the Spec Committee call that we will accept 1.2.7 version.  Please update this API coordinate to point at the 1.2.7 API jar file.

**Comment by kwsutter** [comment]
> @ren-zhijun-oracle Could you also file a PR for updating jakartaee-api pom.xml with the proper Jakarta JSTL version (1.2.5 or 1.2.7, per the earlier comment)?
> 
> https://github.com/eclipse-ee4j/jakartaee-api/pulls

Never mind.  If you are good with the 1.2.7 version, I have updated the pom in jakartaee-api for you via [PR #34](https://github.com/eclipse-ee4j/jakartaee-api/pull/34).  If you decide on an alternate version, then please submit another PR.  Thank you.

**Comment by ren-zhijun-oracle** [comment]
> > @ren-zhijun-oracle Could you also file a PR for updating jakartaee-api pom.xml with the proper Jakarta JSTL version (1.2.5 or 1.2.7, per the earlier comment)?
> > https://github.com/eclipse-ee4j/jakartaee-api/pulls
> 
> Never mind. If you are good with the 1.2.7 version, I have updated the pom in jakartaee-api for you via [PR #34](https://github.com/eclipse-ee4j/jakartaee-api/pull/34). If you decide on an alternate version, then please submit another PR. Thank you.

Hi, I don't know why there is no 1.2.7: https://search.maven.org/artifact/jakarta.servlet.jsp.jstl/jakarta.servlet.jsp.jstl-api/1.2.7/jar, caused by it's in staging phase?  https://search.maven.org/artifact/jakarta.servlet.jsp.jstl/jakarta.servlet.jsp.jstl-api/1.2.5/jar is existing.  can you help?

**Comment by ren-zhijun-oracle** [comment]
> > > @ren-zhijun-oracle Could you also file a PR for updating jakartaee-api pom.xml with the proper Jakarta JSTL version (1.2.5 or 1.2.7, per the earlier comment)?
> > > https://github.com/eclipse-ee4j/jakartaee-api/pulls
> > 
> > 
> > Never mind. If you are good with the 1.2.7 version, I have updated the pom in jakartaee-api for you via [PR #34](https://github.com/eclipse-ee4j/jakartaee-api/pull/34). If you decide on an alternate version, then please submit another PR. Thank you.
> 
> Hi, I don't know why there is no 1.2.7: https://search.maven.org/artifact/jakarta.servlet.jsp.jstl/jakarta.servlet.jsp.jstl-api/1.2.7/jar, caused by it's in staging phase? https://search.maven.org/artifact/jakarta.servlet.jsp.jstl/jakarta.servlet.jsp.jstl-api/1.2.5/jar is existing. can you help?

do you mean I just change it to:  
* Maven coordinates
  * [jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api:jar:1.2.7](https://search.maven.org/artifact/jakarta.servlet.jsp.jstl/jakarta.servlet.jsp.jstl-api/1.2.7/jar)

Then the link will available when 1.2.7 is released from staging state?

File: tags/1.2/_index.md (hunk 7 above)
Comment (bshannon) [reply tags/1.2/_index.md]: Yes, @ren-zhijun-oracle in two places on the line above, the "1.2" should be "1.2.7".

File: tags/1.2/_index.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+Jakarta Standard Tag Library encapsulates as simple tags the core functionality common to many Web applications.
+Jakarta Standard Tag Library has support for common, structural tasks such as iteration and conditionals, tags
+for manipulating XML documents, internationalization tags, and SQL tags. It also provides a framework for integrating
+existing custom tags with Jakarta Standard Tag Library tags.
+
+* [Jakarta Standard Tag Library 1.2 Specification Document](./tags-1.2-spec.pdf) (PDF)
+* [Jakarta Standard Tag Library 1.2 Specification Document](./tags-1.2-spec.html) (HTML)
+* [Jakarta Standard Tag Library 1.2 Javadoc](./apidocs)
+* [Jakarta Standard Tag Library 1.2 Tagdoc](./tagdocs)
+* [Jakarta Standard Tag Library TCK](https://download.eclipse.org/jakartaee/tags/1.2/eclipse-tags-tck-1.2.0.zip)
```
Comment (bshannon) [inline tags/1.2/_index.md]: @ren-zhijun-oracle  One more minor issue (I should've caught this previously), "Library TCK" -> "Library 1.2 TCK".
Thanks.

**Comment by starksm64** [comment]
* On ballot completion
 - [x] The designated specification committee representative adds this final checklist to the main PR.
 - [ ] The specification committee representative merges the specification PRs.
 - [ ] The designated specification committee representative calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job.
 - [ ] The specification project member who created the staging release promotes the specification api jars to maven central. An example release job script can be found here https://wiki.eclipse.org/MavenReleaseScript.
 - [ ] The EMO updates the specification page with the ballot results.
This list goes on the committed spec index page.
 - [ ] The specification project team should go through the merged spec website page to verify all the links are valid.
 - [ ] The specification project team should approve the compatibility request.
 - [ ] The compatible implementation project/vendor MUST send an email tck@eclipse.org for approval of the compatible implementation for trademark usage.
 - [ ] The specification project team should merge any final release branch as appropriate for the branch management for the project.

**Comment by starksm64** [comment]
@ren-zhijun-oracle you should be able to release the api jars to maven central now.

**Comment by ruolli** [comment]
@starksm64  we released JSTL-API 1.2.7 formally to http://central.maven.org/maven2/jakarta/servlet/jsp/jstl/jakarta.servlet.jsp.jstl-api/1.2.7/ . 

Thanks.

**Comment by bshannon** [comment]
@ruolli Please confirm that you've completed the last two items in the [checklist](https://github.com/jakartaee/specifications/pull/22#issuecomment-528190977).

**Comment by ruolli** [comment]
@bshannon Bill, about this item "The compatible implementation project/vendor MUST send an email tck@eclipse.org for approval of the compatible implementation for trademark usage." , should the mail be sent by GlassFish team ? 

As to the item "The specification project team should merge any final release branch as appropriate for the branch management for the project." , the branch had been merged .

**Comment by bshannon** [comment]
@ruolli The message to tck@eclipse.org should come from the lead of the implementation project
for the Jakarta Standard Tag Library; I believe that would be you or @ren-zhijun-oracle.

**Pull request merged**